Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -43,6 +43,7 @@ @interface OWSMessageCell ()
 
 @property (nonatomic) UILabel *dateHeaderLabel;
 @property (nonatomic) OWSMessageTextView *bodyTextView;
+@property (nonatomic) UILabel *quotedTextView;
```

Comment:
We just use a label - we don't need to use data detectors or otherwise interact with the quoted text.